libvast/builtins/formats/json.cpp

```diff
@@ -588,7 +596,10 @@ class default_parser : public parser_base<FieldValidator> {
     // to be used with well formed JSON or truncated JSON. In this case we don't
     // know how to recover. It might be possible to use different parser or our
     // custom logic to try recover as much data as possible.
+    // TODO: revisit if this is needed now that we abort and co_return in
```

Comment:
This would take some time to investigate so i'd rather leave a TODO here. Should i open an issue for it and remove the TODO from the code? Or leave the todo and create issue?

Reply:
I think that we should not be able to run into this anymore now, so the comment should say something along the lines of "likely not needed anymore, need to investigate," and then in the future this should be an assertion.